source/applySpec.js

```diff
@@ -1,12 +1,19 @@
 import _curry1 from './internal/_curry1';
 import apply from './apply';
 import curryN from './curryN';
-import map from './map';
 import max from './max';
 import pluck from './pluck';
 import reduce from './reduce';
 import values from './values';
+import fromPairs from './fromPairs';
+import toPairs from './toPairs';
 
+// Use the built-in map function to avoid issues with specs what include a "map" key
+function mapValues(fn, obj) {
+  return fromPairs(toPairs(obj).map(function(pair) {
+    return [pair[0], fn(pair[1])];
+  }));
+}
```

Comment:
I wonder if we shouldn't use an implementation more like [what's in `map`][ma], but with the native `reduce`:

```js
function mapValues(fn, obj) {
  return keys(obj).reduce(function(acc, key) {
    acc[key] = fn(obj[key]);
    return acc;
  }, {})
}
```
Ramda tries not to build on its external functions too much, preferring somewhat ugly imperative code to improve performance.  We would use `keys` here, as Ramda through 1.0 is supporting older engines that don't necessarily have `Object.keys`.  But if we can avoid `fromPairs`/`toPairs`, it would (presumably) be faster.

   [ma]: https://github.com/ramda/ramda/blob/master/source/map.js#L51-L55